kotlin-guiced.gradle.kts

```diff
@@ -69,10 +70,26 @@ subprojects {
 
         "testCompile"(junitJupiter("junit-jupiter-api"))
         "testCompile"(junitJupiter("junit-jupiter-params"))
+        "testCompile"(group = "com.natpryce", name = "hamkrest", version = "1.5.0.0")
         "testRuntime"(junitJupiter("junit-jupiter-engine"))
         "testRuntime"(create(group = "org.junit.platform", name = "junit-platform-launcher", version = "1.3.1"))
     }
 
+    tasks.withType<KotlinCompile>().configureEach {
+        // Kotlin incremental compilation is enabled by default
+        kotlinOptions {
+            jvmTarget = "1.8"
+
+            freeCompilerArgs += "-Xprogressive"
```

Comment:
This should be `-progressive`.

Reply:
I knew it would be good to have you on this PR. Thanks!

Reply:
We aren't at Kotlin 1.3 for this project yet so this can't be done yet, unfortunately.